charts/emissary-ingress/templates/controller-ingressclass.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.controller.ingressClassResource.enabled -}}
```

Comment:
There is no top level `controller` in your additions to the values. This should be just `.Values.ingressClassResource.enabled` to match your addition to the Values file.

Reply:
> There is no top level `controller` in your additions to the values. This should be just `.Values.ingressClassResource.enabled` to match your addition to the Values file.

Nice find! Thanks!

I pulled latest from upstream master, fixed the `controller` thingie in values, and added info about the ingress class in changelog as well.